utility/mm2_update-master-directory-list

```diff
@@ -76,46 +88,54 @@ def make_file_details_from_checksums(session, diskpath, relativeDName, D):
         d = {}
         checksum_files = []
         for g in globs:
-            checksum_files.extend(
-                glob.glob(os.path.join(diskpath, relativeDName, g)))
+            if files != None:
```

Comment:
I believe you _could_ rewrite this so you don't need to duplicate code
```
file_paths = files.keys() if files else os.listdir(os.path.join(config.get('UMDL_PREFIX', ''), relativeDName))
for f in file_paths:
    if re.compile(g).match(f):
          checksum_files.append(os.path.join(relativeDName,f))
```
Total nick-pick though.